builder-image/image-jdk21-overrides.yaml

```diff
@@ -0,0 +1,40 @@
+schema_version: 1
+
+name: &imgName "jboss-eap-8/eap8-openjdk21-builder-openshift-rhel9"
+# This OSBS Base Image is designed and engineered to be the base layer for
+# Red Hat products. This base image is only supported for approved Red Hat
+# products. This image is maintained by Red Hat and updated regularly.
+from: registry.redhat.io/rhel9-osbs/osbs-ubi9-minimal
+labels:
+    - name: "org.jboss.product"
+      value: "eap80-openjdk21-builder"
+    - name: "com.redhat.component"
+      value: "jboss-eap80-openjdk21-builder-openshift-container"
```

Comment:
The correct brew package is `jboss-eap8-openjdk21-builder-openshift-container`

Reply:
Thanks fixed.